sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
+  waiting. intel/llvm#17260
+- Made `-O0` to be the default optimization level when debug info is enabled
+  through `-g` flag. intel/llvm#16408
+- Uplifted maximum version of SPIR-V that compiler can generate to 1.5.
+  intel/llvm#16626
+- Made compiler embed device library needed for `bfloat16` support into the
+  application (if it is used). This change will allow us to reduce the size
+  of redistributable SYCL RT package by eliminating some files from it.
+  intel/llvm#16729
+- Added a compiler diagnostic (warning) about undefined `SYCL_EXTERNAL`
+  functions used in a module to help catch linking errors earlier.
+  intel/llvm#17346
+- Addressed issue intel/llvm#11531 where the compiler would generate invalid
+  SPIR-V if kernel used arguments of boolean type. intel/llvm#17427
+- Switched to use native `bfloat16` implementation for devices that support it
+  (LNL, PVC), as well as fixed a bug where native implementation won't be used
+  if multiple AOT targets are specified. intel/llvm#17154, intel/llvm#16240,
+  intel/llvm#16494
+- Aligned behavior of `-Wimplicit-float-conversion` with the upstream clang for
+  non-SYCL language modes. intel/llvm#16857
+- Added support for `dynamic_address_cast` on CUDA & HIP backends.
+  intel/llvm#16604
+- Fixed compilation errors when building applications that use `nearbyint` and
+  `rint` for HIP targets. intel/lllvm#16373
+- Improved check for unsupported data types to actually rely on target
+  information instead of hardcoded knowledge. For example, this allows 128-bit
+  integeres to be used in device code when targeting CUDA backend.
+  intel/llvm#17036
+- Fixed hangs on AMD and crashes on NVIDA when `atomic_ref` is used with
+  `work_item` memory scope. intel/llvm#16172
+- Fixed `-fcuda-short-ptr` flag causing compilation errors. Its use will still
+  result in a warning that some implicitly linked object is not compiled with
+  that flag (namely some of our built-in libraries), but it shouldn't be a
+  problem because those libraries don't operate on pointers. intel/llvm#15642
+- Fixed intel/llvm#15852 where compilation with `-mlong-double-64` would still
+  result in error that 128 double is not supported by a target. intel/llvm#16441
+- Fixed a bug that linking static libraries with SYCL code in them using
+  `-l:libname.a` spelling would ignore device code from those libraries.
+  intel/llvm#17149
+- Fixed a bug where having a pure virtual function marked as device one would
+  cause unresolved symbol errors emitted by device compiler on Windows.
+  intel/llvm#16231
+- Fixed a bug where having two kernels (one annotated with
+  `reqd_work_group_size` attribute/property and another without it) together
+  with `-fsycl-device-code-split=off` would cause runtime error about
+  mismatched work-group size. intel/llvm#16236
+- Fixed debug information for kernels that use global offest on HIP & CUDA
+  backends. intel/llvm#16963
+
+### Other changes in SYCL Library
```

Comment:
@intel/unified-runtime-reviewers, @intel/llvm-reviewers-cuda, tagging you to have a glance over this generic section